crates/goose/src/config/base.rs

```diff
@@ -116,14 +116,18 @@ enum SecretStorage {
 // Global instance
 static GLOBAL_CONFIG: OnceCell<Config> = OnceCell::new();
 
+pub fn get_config_dir() -> PathBuf {
```

Comment:
@jamadeo this would radically change how cli behaves wouldn't it? ie people that launch things from whatever shell env they are in, will no longer get the same environment until they migrate things over?